src/main/java/com/andersentask/bookshop/console/Commands.java

```
@@ -91,73 +69,49 @@ public List<Book> getSortedBooks(BookSort bookSort) {
      * completed can't be changed. To be completed, order should have all of it books available
-     * Returns result of operation as ENUM:
-     * WRONG_ORDER_ID => if the input can be parsed to Long, but there is no such ID for order
-     * ORDER_STATUS_CAN_NOT_BE_UPDATED => if the logic is not met
-     * STATUS_UPDATED => if the order status was successfully updated
      *
      * @param id          id of order and should be got from user
      * @param orderStatus can be completed, canceled or in_proccesing and should be got from user
-     * @return the status of method completion as ENUM
      */
-    public ResultOfOperation.ChangeStatusOfOrderIncludingBooksCheck changeStatusOfOrderIncludingBooksCheck(Long id, OrderStatus orderStatus) {
-        Optional<Order> optionalOrder = appContextConfig.getOrderService().getOrderById(id);
-        ResultOfOperation.ChangeStatusOfOrderIncludingBooksCheck resultOfMethod
-                = ResultOfOperation.ChangeStatusOfOrderIncludingBooksCheck.WRONG_ORDER_ID;
-
-        if (optionalOrder.isPresent()) {
-            Order order = optionalOrder.get();
-            OrderStatus previousStatus = order.getOrderStatus();
-            if (orderStatus.equals(OrderStatus.COMPLETED)) {
-                if (appContextConfig.getBookService().allBooksAreAvailable(order.getBooksInOrder())) {
-                    appContextConfig.getOrderService().changeStatusOfOrder(id, orderStatus);
-                }
-            } else appContextConfig.getOrderService().changeStatusOfOrder(id, orderStatus);
-
-            if (previousStatus.equals(order.getOrderStatus())) {
-                resultOfMethod = ResultOfOperation.ChangeStatusOfOrderIncludingBooksCheck.ORDER_STATUS_CAN_NOT_BE_UPDATED;
-            } else resultOfMethod = ResultOfOperation.ChangeStatusOfOrderIncludingBooksCheck.STATUS_UPDATED;
-        }
-        return resultOfMethod;
+    public void changeStatusOfOrderIncludingBooksCheck(Long id, OrderStatus orderStatus) {
```

if the status of the order was not changed, we should throw an exception
we somehow should trace, if the status is really changed

---

to discuss